src/Cron.php

```diff
@@ -131,4 +156,47 @@ public static function isValid(string $expression): bool
     {
         return CronExpression::isValidExpression($expression);
     }
+
+    public static function fromJson(string $data): Frequency
+    {
+        $data = json_decode($data, true);
+        $start = null;
+        $end = null;
+
+        if (isset($data['start'])) {
+            $start = new DateTime($data['start']);
+            unset($data['start']);
```

Comment:
Why unset? `$data` isn't passed on afterwards anyway.

Reply:
Due to the use of `current()`.